tensorflow/core/kernels/depthwise_conv_ops_test.cc

```diff
@@ -102,8 +102,8 @@ TEST_F(DepthwiseConvOpTest, DepthwiseConvHalfCpu) {
   Run<Eigen::half>(Device::CPU);
 }
 
-#ifdef GOOGLE_CUDA
-TEST_F(DepthwiseConvOpTest, DepthwiseConvFloatGpu) { Run<float>(Device::GPU); }
+#if GOOGLE_CUDA || TENSORFLOW_USE_ROCM
+// TEST_F(DepthwiseConvOpTest, DepthwiseConvFloatGpu) { Run<float>(Device::GPU); }
```

Comment:
Q: Why is this commented out?

Reply:
that is a mistake.  let me fix and push out the updated commit

Reply:
done